com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2021.2-to-2022.1.md

```diff
@@ -1,5 +1,10 @@
-# Upgrading HDRP from 2021.2 to 2022.1
+# Upgrading HDRP from 2021.1 to 2021.2
 
 In the High Definition Render Pipeline (HDRP), some features work differently between major versions. This document helps you upgrade HDRP from 12.x to 13.x.
 
+## Color Grading
+
+Starting from HDRP 13.x, HDRP will use ACEScg as a default color space to perform color grading even when using a non-ACES  if  unlike the previously used sRGB color space.  It is possible to go back to sRGB in the Global Settings Asset.
```

Comment:
Thank you so much for including the upgrade guide and what's new updates in your PR! : D

Proposed revision:
"ACEScg is now the default color space for color grading for all HDRP projects.  When you upgrade a project from an earlier version of HDRP, that project's color space automatically changes to ACESsg. To revert your project's color space to sRGB, navigate to **Edit > Project Settings > Graphics > [HDRP Global Settings](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@13.0/manual/Default-Settings-Window.html)** > and "

It isn't actually clear to me where users can change their color space setting back to sRGB--I've looked this PR, in the editor, and the HDRP docs. Maybe I am missing something? 

Also, I assume when a user opens a project created with an earlier version of HDRP in a 13.x version of HDRP, the Unity engine automatically changes the color space of that old project from sRGB to ACES.
If I understood incorrectly, let me know.

Reply:
@FrancescoC-unity Was tracking the thumb ups, saw this one wasn't marked. Thoughts?

Reply:
I changed this in the newer commits